src/journal/lookup3.c

```diff
@@ -787,15 +792,15 @@ void driver1()
     h = hashlittle(&buf[0],1,h);
   }
   time(&z);
-  if (z-a > 0) printf("time %d %.8x\n", z-a, h);
+  if (z-a > 0) printf("time %ld %.8x\n", z-a, h);
```

Comment:
hmm, if we wanted to do this fully correctly, we'd define TIME_FMT that maps to the correct format string for time_t... We do it like that for uid_t → UID_FMT, pid_t → PID_FMT and so on.

Reply:
I tried using the function `difftime()` and used `lf` as the string format for doubles to have it as close as it was before. If that's what you want it?